Backpack/Tests/SnapshotTests/BPKCalendarSnapshotTest.swift

```diff
@@ -315,7 +315,7 @@ final class BPKCalendarSnapshotTest: XCTestCase, BPKCalendarDelegate {
         
         if day == 18 {
             return isShowingPrices
-            ? BPKCalendarPriceLabelCellData(price: "-", labelStyle: .noData)
+            ? BPKCalendarPriceLabelCellData(icon: BPKIcon.makeSmallTemplateIcon(name: .search))
```

Comment:
Checking the snapshot but I think you have an option here because now many date are blue it seems like